tests/test_ms_machine_id_for_TR.py

```diff
@@ -0,0 +1,26 @@
+from ctrlibrary.threatresponse.enrich import enrich_observe_observables
+
+
+def test_ms_machine_id_doesnt_support_by_TR(module_headers):
+    """Perform testing for verifying that 'ms_machine_id' type doesn't
+    support by TR.
+
+    ID: CCTRI-1957-f67174f1-6932-4247-bad4-807b5c31e486
+
+    Steps:
+        1. Send request to enrich observe observable endpoint
+
+    Expectedresults:
+        1. Check that errors in response body doesn't contain date from
```

Comment:
This is not too important, but you probably meant not `date`, but `date` here. And maybe it's even better to formulate it like this: Check that response body doesn't contain data. 
Because errors definitely cannot contain data inside themselves.